spec/models/rehearsal_spec.rb

```diff
@@ -19,15 +19,13 @@
   end
 
   it 'can be read by the user to whom this rehearsal belongs' do
-    user = FactoryBot.create(:user)
-    rehearsal = FactoryBot.create(:rehearsal, user: user)
-
-    expect(Rehearsal.where(user_id: user.id)).to include(rehearsal)
+    should belong_to(:user)
   end
 
   it 'cannot be read by a user to whom this rehearsal does not belong' do
     FactoryBot.create(:user, id: 55)
```

Comment:
Again, you're testing the very definition of a `belongs_to` relationship here. This is already covered by `should belong_to(:user)`. 

Maybe you want to look into what a `has_many`/`belongs_to` means in terms of database column setup and resulting queries? This is solely related to database querying, there is no authorization to be tested at play here.

Reply:
I'm confused.

I have different types of admins. One admin type can view, for example, lists of users and lists of all exercises. 

Previously, I had bugs whereby normal users could view other user's private exercises. (If they simply typed in the url `/exercises/:private_exercise_id`). It is certainly theoretically possible for a normal user to view the private rehearsal of another user. I setup the `CanCanCan` Abilities to prevent this.

Why wouldn't it be possible to have a particular `rehearsal` BOTH `belong_to` a `user`-- and also be visible to other users? 

Therefore, this test is meaningful. These types of tests are exactly what helped me to spot holes in my admin / `CanCanCan` abilities in the exercises model.